migrations/20200831143314_endpointNameLength.js

```diff
@@ -0,0 +1,46 @@
+/*****
```

Comment:
unsure what file naming scheme the migration uses :thinking: `migrate:create` give me `20200831143314...js where as other migrations are some different numbering scheme.